### PR TITLE
Add a sample project for running Spark Streaming with ScyllaDB

### PR DESCRIPTION
**Pull request by iravid**

### CODE REVIEW

File: scylla-and-spark/streaming-into-scylla/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Next, create the keyspace and table:
+```shell
+./create-tables.sh
+```
+The `quotes` keyspace and table will be created, along with a `quotes_by_day` materialized view for running the statistics endpoint.
+
+Finally, run the service:
+```shell
+./run.sh
```
[the remark this answers is not in the document]
Comment (tzach) [reply scylla-and-spark/streaming-into-scylla/README.md]: I'm struggling with the following error
```
[error] java.lang.RuntimeException: Could not create directory /home/tzach/project/iravid-scylla-code-samples/scylla-and-spark/streaming-into-scylla/target/streams/$global/assembly/$global/streams
```

[the remark this answers is not in the document]
Comment (iravid) [reply scylla-and-spark/streaming-into-scylla/README.md]: That's weird. Can you try creating it manually with `mkdir -p`?

[the remark this answers is not in the document]
Comment (tzach) [reply scylla-and-spark/streaming-into-scylla/README.md]: @iravid should the $global be defined somehow?

[the remark this answers is not in the document]
Comment (iravid) [reply scylla-and-spark/streaming-into-scylla/README.md]: Hmm no, you should escape it

**Pull request merged**